helpers.js

```diff
@@ -0,0 +1,22 @@
+exports.validMessage = function(message) {
+  let result = { isError: false, errorMsg: "" };
+  let invalidSymbols = [" ", ".", ","];
+  if (
+    message.length <= 3 ||
+    message.length > 30 ||
+    invalidSymbols.findIndex(item => message.includes(item)) > -1
+  ) {
+    result.errorMsg =
+      "Название города должно содержать одно слово, \nбез пробелов и знаков препинания, \nот 3 до 30 символов.";
+    result.isError = true;
+  }
+  return result;
+};
+
+exports.firstSymbolToUpperCase = function(str) {
+  return str[0].toUpperCase() + str.slice(1);
+};
+
+exports.getRandomInt = function(max) {
```

Comment:
Int – is not common word for js community, because we don't have ints, technically all numbers in js are `float`, so use just `number` 
```suggestion
exports.getRandomNumber = function(max) {
```

Reply:
Ok. I fix that.

Reply:
```suggestion
function getRandomNumber(max) {
  return Math.floor(Math.random() * Math.floor(max));
}
```
Fixed it